client/state/sites/selectors/is-jetpack-site.js

```diff
@@ -1,18 +1,36 @@
 import getRawSite from 'calypso/state/selectors/get-raw-site';
+import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
+import { AppState } from 'calypso/types';
 
 /**
  * Returns true if site is a Jetpack site, false if the site is hosted on
  * WordPress.com, or null if the site is unknown.
  *
- * @param  {object}   state  Global state tree
+ * @param  {AppState} state  Global state tree
  * @param  {?number}   siteId Site ID
- * @returns {?boolean}        Whether site is a Jetpack site
+ * @param  {boolean}  considerStandaloneProducts Whether to consider sites with Jetpack standalone plugins installed
+ * @returns {boolean | null} Whether site is a Jetpack site
  */
-export default function isJetpackSite( state, siteId ) {
+export default function isJetpackSite( state, siteId, considerStandaloneProducts = true ) {
```

Comment:
This is not a huge deal, but boolean parameters are not ideal because they can be hard to read at call-sites. `isJetpackSite( state, 12345, false )` is not very semantic.

Two suggestions:

1. Make the parameter a string/constant. `isJetpackSite( state, 12345, 'exclude-standalone-products' )`
2. Make the parameter an object with an optional key. `isJetpackSite( state, 12345, { includeStandaloneProducts: false } )`.

Reply:
Yes, I like the object approach. Done. Thanks